tests/smoke/test_tzkt_api.py

```diff
@@ -58,5 +58,5 @@ def test_base_url(ConfigParser, args):
         )
     )
     args.initial_cycle = 100
-    args.api_base_url = 'https://api.carthage.tzkt.io/v1/'
+    args.api_base_url = 'https://api.tzkt.io/v1/'
```

Comment:
This is the mainnet URL and above is the testnet URL from giganode. this doesn't seem to fit